Resources/public/js/models/ez-locationmodel.js

```diff
@@ -303,6 +303,17 @@ YUI.add('ez-locationmodel', function (Y) {
         },
 
         /**
+         * Returns if the location is a root one
+         * 
+         * @method _isRootLocation
+         * @return {boolean}
+         * @public
+         */
+        _isRootLocation: function() {
```

Comment:
public method should not start by `_` sorry… I should have mentioned it.
You don't need the `@public`